Formula/fcct.rb

```diff
@@ -0,0 +1,51 @@
+class Fcct < Formula
+  desc "Fedora CoreOS Config Transpiler"
+  homepage "https://github.com/coreos/fcct"
+  url "https://github.com/coreos/fcct/archive/v0.5.0.tar.gz"
+  sha256 "50912f67ecc2da62b45f597522a685350422998d5840a18828fc9505a5dc51db"
+  head "https://github.com/coreos/fcct.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GO111MODULE"] = "on"
+    ENV["GOFLAGS"] = "-mod=vendor"
```

Comment:
This seems a bit odd. Why not pass it directly to the `go build` command?

Reply:
I reused the build instructions that the original `fcct` build script provided – I will double check and update the pull request accordingly.